src/scrapers/movie/IMDB.cpp

```diff
@@ -197,6 +228,24 @@ void IMDB::onSearchIdFinished()
     emit searchDone(results, {});
 }
 
+QVector<ScraperSearchResult> IMDB::parseSearchAdult(QString html)
+{
+    QVector<ScraperSearchResult> results;
+
+    QRegExp rx("<a href=\"/title/([t]*[\\d]+)/[^\"]*\"\\n>([^<]*)</a>\\n.*(?: \\(I+\\) |>)\\(([0-9]*).*\\)");
```

Comment:
```suggestion
    QRegExp rx("<a href=\"/title/(tt[\\d]+)/[^\"]*\"\\n>([^<]*)</a>\\n.*(?: \\(I+\\) |>)\\(([0-9]*).*\\)");
```

IMDb IDs always start with `tt` :-)

Reply:
The suggestion is tested and works well after recompilation (4082e89). Thank you!